enterprise/models.py

```diff
@@ -2688,7 +2689,11 @@ class SystemWideEnterpriseUserRoleAssignment(EnterpriseRoleAssignmentContextMixi
     .. no_pii:
     """
 
-    role_class = SystemWideEnterpriseRole
+    role = models.ForeignKey(
```

Comment:
the FK method is one I'm a little more familiar with than (what seems to be?) assigning the object to the field, could I get a tl:dr on what the `role_class` was doing before this?

Reply:
oh just saw your description- nvm 😄

Reply:
ah this was answered via dm, but role_class is a field value picked up by the base class (class inheritance ultimately leads back to here: https://github.com/edx/edx-rbac/blob/master/edx_rbac/models.py#L24).
When role_class is present, it adds a field named `role` to the model that is a foreign key to the model associated with the value of role_class here. In much simpler verbiage: it does exactly what I'm doing in this pr. When role is present, it does nothing, because it doesn't need to add a foreign key field where we already have that exact foreign key field.